eng/ci/templates/official/jobs/consolidate-cli-artifacts.yml

```diff
@@ -98,7 +100,7 @@ jobs:
         targetType: filePath
         filePath: '$(Build.SourcesDirectory)/eng/scripts/generate-msi-files.ps1'
         arguments: '-ArtifactsPath "$(Pipeline.Workspace)/staging/func-cli"'
```

Comment:
@aishwaryabh or @soninaren - do you know why we only generate MSI for x architectures? Can we generate an MSI build for arm?